data/campaigns/Heir_To_The_Throne/scenarios/06_The_Siege_of_Elensefar.cfg

```diff
@@ -232,325 +232,381 @@
         [/message]
     [/event]
 
+
+
+# Thieves’ Guild
+# Handle the following cases:
+#  1) Enemies defeated before turn 4, four thieves join the revelry in town
```

Comment:
Call it "future proofing" .. what if someone adds a earlier scenario and suddenly Konrad actually comes to Elensefar with sufficient force to actually beat both leaders in 3 turns?